cves/CVE-2016-2162.yml

```diff
@@ -205,4 +223,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
What can be done to prevent a vulnerability like this from happening again?

Reply:
Being an oversight, including more thoughts on security in all phases of design and development would help prevent this type of issue.